Remove docs for examples, tutorials, streams, flows, etc.

anwar6953 opened the pull request:
Remove docs for examples, tutorials, streams, flows, etc.

https://builds.cask.co/browse/CDAP-DQB458-7

Commit:
Fix MD5hash for integrations

Commit:
Update MD5 Hash

Commit:
Update Hash

Commit:
Remove examples and tutorials

Commit:
Remove examples manual

Commit:
Update examples-manual reference

Commit:
remove examples manual

Commit:
Remove examples

Commit:
Update examples

Commit:
Remove some references to examples.

Commit:
Remove documentation about examples.

Commit:
Fix builds

Commit:
Remove references to example docs.

Commit:
Remove tutorials and cdap apps, packs.

Commit:
Remove references to streams. Remove additional references to examples.

Commit:
Remove Ingest command

anew commented on cdap-docs/developer-manual/source/advanced/configuring-resources.rst:
```diff
@@ -84,7 +84,7 @@ in the ``default`` namespace, and would like to set the memory used by its YARN
           -H 'Content-Type: application/json' -d '{ "task.mapper.system.resources.memory": 2048 }'
 
-- To set the memory used by the mapper of the *PurchaseHistoryBuilder* MapReduce job, you would use::
+- To set the memory used by the mapper of the *MyMapReduce* MapReduce job, you would use::
```
is this correct? It seems the same as setting it for all MapReduce

anwar6953 replied:
The example is setting this argument on just the one MapReduce job (line 96).

anew commented on cdap-docs/developer-manual/source/advanced/configuring-resources.rst:
```diff
@@ -111,13 +111,13 @@ in the ``default`` namespace, and would like to set the memory used by its YARN
     .. tabbed-parsed-literal::
 
-      $ curl -w"\n" -X PUT "http://example.com:11015/v3/namespaces/default/apps/PurchaseHistory/flows/PurchaseFlow/preferences" \
+      $ curl -w"\n" -X PUT "http://example.com:11015/v3/namespaces/default/apps/MyApp/flows/PurchaseFlow/preferences" \
```
should this section be removed entirely? There are no more flows.

anwar6953 replied:
removed.

anew commented on cdap-docs/developer-manual/source/building-blocks/mapreduce-programs.rst:
```diff
@@ -376,14 +376,7 @@ MapReduce and Resources
-   :language: java
-   :lines: 47-57
-   :append: ...
+as part of the configuration. They can also be set at runtime through the use of runtime arguments.
```
should this be replaced by a literal piece of code, instead of deleting it? Otherwise, it is not clear what configurations are required.

anew commented on cdap-docs/developer-manual/source/building-blocks/services.rst:
```diff
@@ -234,22 +222,4 @@ Both the number of service threads and the thread keep-alive time can be altered
-  :ref:`WordCount example <examples-word-count>`, where the class ``RetrieveCountsHandler``
-  retrieves a variety of statistics from datasets depending on the path supplied. 
-
-- Almost all of the :ref:`how-to guides <guides-index>` demonstrate the use of services.
```
aren't we keeping the guides?

anwar6953 replied:
Yes, but this statement is no longer true. A majority of them don't demonstrate services.

anew commented on cdap-docs/developer-manual/source/building-blocks/spark-programs.rst:
```diff
@@ -73,14 +73,6 @@ When a Spark program is configured, the resource requirements for both the Spark
-the ``PageRankSpark``, the amount of memory is specified:
-
-.. literalinclude:: /../../../cdap-examples/SparkPageRank/src/main/java/io/cdap/cdap/examples/sparkpagerank/SparkPageRankApp.java
-   :language: java
```
similar to above, can we have literal code to show the properties that need to be set?

anew commented on cdap-docs/developer-manual/source/building-blocks/transaction-system.rst:
```diff
@@ -143,15 +143,8 @@ By default, CDAP will start an implicit transaction for these methods:
-   :lines: 118-140
-   :dedent: 2
+CDAP implicitly starts a transaction for this handler method, and the handler can rely
+on the transactional consistency of the data it reads from the dataset.
```
Is this still correct? Or did we change to make all transactions explicit in 6.0? @chtyim might know

anew commented on cdap-docs/developer-manual/source/building-blocks/transaction-system.rst:
```diff
@@ -143,15 +143,8 @@ By default, CDAP will start an implicit transaction for these methods:
+CDAP implicitly starts a transaction for this handler method, and the handler can rely
+on the transactional consistency of the data it reads from the dataset.
 
 For flowlet process methods, this starting of implicit transactions cannot be disabled,
```
no more flows :)

anew commented on cdap-docs/developer-manual/source/testing/testing.rst:
```diff
@@ -191,8 +191,7 @@ Strategies in Testing MapReduce Programs
 ========================================
 In a fashion similar to `Strategies in Testing Flows`_, we can write
```
noe more flows?

anew commented on cdap-docs/examples-manual/source/index.rst:
```diff
@@ -4,41 +4,16 @@
 
 
 .. |examples| replace:: **Examples:**
 .. _examples: examples/index.html
```
should this go?

anew commented on cdap-docs/examples-manual/source/index.rst:
```diff
@@ -4,41 +4,16 @@
 .. |guides| replace:: **How-To Guides:**
 .. _guides: how-to-guides/index.html
 
 - |guides|_ Designed to be completed in 15-30 minutes, these guides provide quick, hands-on
```
seems that - since guide are the only thing that's left - we can move the contents of how-to-guide/index.rst here. Otherwise this page becomes a moot point, all it has is a pointer to another page?

anew reviewed (commented):
Please see my comments. 

I can only speak to the changes I see here, It is hard to tell whether more changes are required, 

Also, please update the copyrights of all the files that changed

Commit:
Address comments.

anwar6953 commented:
I addressed some of the comments. I will address the remainder in a follow-up PR, since this PR fixes the docs quick build and other already-opened PRs are waiting on the docs quick build to be working.

CuriousVini reviewed (approved):
lgtm in order to have green docs build

anew commented:
sgtm
